lib/dao/worker_dao.es6.js

```diff
@@ -44,7 +44,7 @@ foam.CLASS({
           of: this.of,
         }, this.localCtx);
         this.propertyIndexGroups
-            .forEach((index) => cache.addPropertyIndex.apply(cache, index));
+            .forEach((index) => cache.addPropertyIndex(...index));
```

Comment:
Actually still not covered in https://codecov.io/gh/GoogleChromeLabs/confluence/src/a14a56ee83584e1116bc709dcb8278d2fbcc967c/lib/dao/worker_dao.es6.js with those tests run.

Reply:
Changing the similar case in indexed_dao.es6.j to `this.addPropertyIndex(index)` makes tests fail, so that confirms that this code wasn't broken, which I suspected because of the singular `index` being an array.